convert2rhel/backup/packages.py

```diff
@@ -177,9 +177,17 @@ def restore(self):
         loggerinst.task("Rollback: Install removed packages")
         if not self._backedup_pkgs_paths:
             loggerinst.warning("Couldn't find a backup for %s package." % ",".join(self.pkgs))
-            return
+            raise exceptions.CriticalError(
+                id_="FAILED_TO_INSTALL_PACKAGES",
+                title="Couldn't find package backup",
+                description=(
+                    "While attempting to roll back changes, we encountered "
+                    "an unexpected failure while we cannot find a package backup."
+                ),
+                diagnosis="Couldn't find a backup for %s package." % ",".join(self.pkgs),
```

Comment:
We could use utils.format_sequence_as_message() here to output the phrase in the correct plural form. But we can update that later.

Reply:
Changed, thanks for the tip :)